.github/workflows/slack-cli-generate-docs.yaml

```diff
@@ -0,0 +1,43 @@
+name: Generate Slack CLI Docs
+
+on:
+  workflow_dispatch:
```

Comment:
👁️‍🗨️ thought: Adding on "version" tags might be interesting! Otherwise we will need to add this to our release runbooks.

📣 ramble: So open to saving this for follow ups of course.